minecode/tests/test_debian.py

```diff
@@ -332,20 +344,20 @@ def test_parse_packages_from_installed_status(self):
 
 class DebianLSLRTest(BaseDebianTest):
 
-    @expectedFailure
     def test_DebianDirectoryIndexVisitor_from_debian(self):
         uri = 'http://ftp.debian.org/debian/ls-lR.gz'
-        test_loc = self.get_test_loc('debian/lslr/ls-lR_debian.gz')
+        temp_gz_location = self.get_tmp_gz_file('./testfiles/debian/lslr/ls-lR_debian')
```

Comment:
These tests are failing because the incorrect test file locations are being passed: https://github.com/nexB/purldb/actions/runs/4289282248/jobs/7472161807#step:6:1289

```suggestion
        test_loc = self.get_test_loc('debian/lslr/ls-lR_debian')
        temp_gz_location = self.get_tmp_gz_file(test_loc)
```